src/Diagrams/Backend/Postscript.hs

```diff
@@ -24,14 +24,16 @@
 -- type in the diagram type construction
 --
 -- > d :: Diagram Postscript
--- > d = ...
+-- > d = square 10
 --
 -- and render giving the @Postscript@ token
 --
--- > renderDia Postscript (PostscriptOptions "file.eps" (Width 400) EPS) d
+-- > renderDia' d (PostscriptOptions "file.eps" (mkWidth 400) EPS)
 --
 -- This IO action will write the specified file.
```

Comment:
It might be nice to mention that `renderDia` can also be used in order to generate an in-memory `ByteString` rather than writing to a file.